kairos-crypto/src/lib.rs

```diff
@@ -20,5 +20,13 @@ pub trait CryptoSigner {
         signature_bytes: U,
     ) -> Result<(), CryptoError>;
 
+    #[cfg(feature = "tx")]
+    fn verify_tx(&self, tx: kairos_tx::asn::Transaction) -> Result<(), CryptoError>;
+    #[cfg(feature = "tx")]
+    fn sign_tx_payload(
+        &self,
+        payload: kairos_tx::asn::SigningPayload,
+    ) -> Result<kairos_tx::asn::Transaction, CryptoError>;
+
```

Comment:
I don't love feature gated parts of traits.
Generally they should be separated out into a separate trait.

I also don't like having a trait with a single impl, but these issues can be addressed in followup at a much latter date.

Reply:
Created follow up task - https://github.com/cspr-rad/kairos/issues/86.